### PR TITLE
Add LockerDome amp-ad support

### PR DESCRIPTION
**Pull request by pskupinski**
[LockerDome](https://lockerdome.com) integration for amp-ad tag.

### CODE REVIEW

**Comment by pskupinski** [comment]
The corporate CLA was signed and accepted.  It now appears under the "Corporate Agreements" section of https://cla.developers.google.com/clas.

I didn't sign it myself, but am covered by it.

Please have the bot run the CLA check again.

**Comment by pskupinski** [comment]
Hey @zhouyx,

Sorry to bother you, but since you appear to be a member of the team that reviews amp-ad pull requests, would it be possible to put this pull request on your team's radar.

I put in a good amount of time reviewing other amp-ad pull requests that have been reviewed to make sure I'm meeting your team's quality expectations with this pull request.

Thanks.

**Comment by zhouyx** [comment]
Hi @pskupinski Thanks for the Pull Request and thanks for letting me know! I'll be reviewing this soon

File: ads/lockerdome.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ */
+
+import {validateData, loadScript} from '../3p/3p';
+
+/**
+* @param {!Window} global
+* @param {!Object} data
+*/
+export function lockerdome(global, data) {
+  validateData(data, ['slot']);
```
Comment (zhouyx) [inline ads/lockerdome.js]: to verify with you, `data-slot` is required not optional right?

File: ads/lockerdome.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+<amp-ad width="300" height="250"
+    type="lockerdome"
+    data-slot="10238273616545382">
+</amp-ad>
+```
+
+## Configuration
+
+Supported parameters:
+- width
```
Comment (zhouyx) [inline ads/lockerdome.md]: There's no need to specify `width` and `height` here, as they are supported for all `amp-ad`, not some special ad type.

File: ads/lockerdome.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+- height
+- data-slot
```
Comment (zhouyx) [inline ads/lockerdome.md]: same question: `data-slot` is required right? if so please indicate so

File: ads/lockerdome.js (hunk 1 above)
Comment (pskupinski) [reply ads/lockerdome.js]: Yes, `data-slot` is required

**Comment by pskupinski** [comment]
Review comments have been addressed

**Comment by zhouyx** [comment]
Looks great! Thanks for the Pull Request. Merged

**Pull request merged**